buildSrc/src/main/java/se/nbis/lega/deployment/test/VerifyTask.java

```diff
@@ -0,0 +1,51 @@
+package se.nbis.lega.deployment.test;
+
+import static se.nbis.lega.deployment.cluster.Machine.LEGA_PRIVATE;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Properties;
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.TaskAction;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class VerifyTask extends TestTask {
+
+    public VerifyTask() {
+        super();
```

Comment:
There's `this.dependsOn("upload");` in `IngestFileTask`. Don't you need it here?